VogtPayroll3/PayrollManager.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace VogtPayroll3
+{
+    class PayrollManager
+    {
+        public void Run()
+        {
+            Display display = new Display();
+            List<Employee> empList = new List<Employee>();
+            Dictionary<int, Employee> dictionary = new Dictionary<int, Employee>();
+
+            //empList = display.PrintMenu();
+            //DisplayEmployeeInformationDictionary(empList);
+
+            dictionary = display.PrintMenu();
+            DisplayEmployeeInformationDictionary(dictionary);
+
+
+        }
+
+        public void DisplayEmployeeInformation(List<Employee> empList)
+        {
+            foreach (var employee in empList)
+            {
+                Console.WriteLine(" ");
+                Console.WriteLine("Name: " + employee.FirstName + " " + employee.LastName);
+                Console.WriteLine("Employee ID: " + employee.EmpID);
+                Console.WriteLine("Gross Pay: " + employee.GetGrossPay());
+                Console.WriteLine("Deductions: " + employee.GetDeductions());
+                Console.WriteLine("Net Pay: " + employee.GetNetPay());
+            }
+        }
+
+        public Dictionary<int, Employee> RemoveUserInDictionary(Dictionary<int, Employee> dictionary, int empID)
+        {
```

Comment:
All of these user manipulation classes should be inside of the `PayrollManager`.

You're managing the payroll users here in `Program.cs`, but you already have a class designated for that. `Program.cs` shouldn't need to know/care how to add an employee to the payroll. Therefore, move these to `PayrollManager`